GitCommands/Git/GitModule.cs

```diff
@@ -1926,14 +1926,14 @@ private string GetNextRebasePatch()
         private static readonly Regex HeadersMatch = new Regex(@"^(?<header_key>[-A-Za-z0-9]+)(?::[ \t]*)(?<header_value>.*)$", RegexOptions.Compiled);
         private static readonly Regex QuotedText = new Regex(@"=\?([\w-]+)\?q\?(.*)\?=$", RegexOptions.Compiled);
 
-        public bool InTheMiddleOfInteractiveRebase()
-        {
-            return File.Exists(GetRebaseDir() + "git-rebase-todo");
-        }
+        private string RebaseTodoFilePath => GetRebaseDir() + "git-rebase-todo.backup";
+        private string CurrentFilePath => GetRebaseDir() + "stopped-sha";
```

Comment:
Could we use constants for strings?

Reply:
In which way?

Reply:
In this case the string is used only once (related to this routine) and the statement is is the accessor so it is used as a constant with a dynamic part.
I lean on not having a constant here, as it adds another layer. (Me and @ivangrek has had that discussion in #8628.) At the same time I see the point when seeing a stray string in code.

In a wider usage, "git-rebase-todo" is hardcoded also in GitUI\Editor\FileViewerInternal.cs SetHighlightingForFile() to highlight in the editor. This file does not need to be syntax highlighted what I see.

To use constants for files GE uses should be a TODO.